contrib/de_DE/LC_MESSAGES/gimp.po

```diff
@@ -21,19 +21,23 @@ msgstr ""
 "dngettext:2,3\n"
 "X-Poedit-Basepath: .\n"
 
-#: gimp.lua:189
+#: gimp.lua:194
 #, lua-format
 msgid "Export Image %i/%i"
 msgstr "Exportiere Bild %i/%i"
 
-#: gimp.lua:194
+#: gimp.lua:242
+msgid "Unable to move edited file into collection. Leaving it as %s"
+msgstr "Die bearbeitete Datei kann nicht in die Sammlung aufgenommen werden. Bearbeitet Datei ist %s"
```

Comment:
Better: "Die bearbeitete Datei kann nicht in die Sammlung aufgenommen werden. Bearbeitete Datei ist %s"